.github/workflows/lint-bot.yml

```
@@ -21,6 +21,12 @@ on:
         description: "Only lint changed files"
         type: boolean
         default: false
+      commit-message:
+        required: true

```

I guess there is no need for having it required

```suggestion
        required: false
```